packages/rollup-plugin-polyfills-loader/rollup-plugin-polyfills-loader.js

```diff
@@ -72,7 +73,12 @@ function rollupPluginPolyfillsLoader(pluginOptions = {}) {
         let preloaded = [];
         for (const entrypoint of entrypoints) {
           preloaded.push(entrypoint.importPath);
-          preloaded.push(...entrypoint.chunk.imports);
+
+          // js files (incl. chunks) will always be in the root directory
```

Comment:
rollup outputs a flat directory by default, but you can actually output into directories

Reply:
yeah... ok to fix/improve once we have a need for it?

Reply:
it would be a breaking change so preferably not